base/exports.jl

```diff
@@ -642,6 +642,7 @@ export
     isbits,
     isequal,
     isimmutable,
```

Comment:
You could move this export to where the function is defined in `deprecated.jl` such that we don't forget to remove it once the function definition is removed (that has happened before).

Reply:
export moved to `deprecated.jl` in 28f7f9f